README.md

```diff
@@ -1,7 +1,95 @@
 # BestFlow
 We are going to fix bad chatbots!
 
-Hosted at: https://bestflow.netlify.app/
+Diagnostic tool for chatbot transcripts that help retailers identify poor or strong questions and intents through an easily digestible and intuitive display.
 
-This project uses:
-React, Axios, Node, Tailwind, MongoDB, Express and more :D
+We aim to:
+- Reduce time spent on chatbot optimization
+- Reduce bad chatbot intents
+- Display an average of 30% of diagnostic problems in any transcripts’ flow
```

Comment:
Was this from back in September? I'm not saying it's bad to have quantified benefits I'm just wondering where we calculated this.

Reply:
I think we can re-visit our concrete benefits during our presentation planning and update the readme